### PR TITLE
Add new color palette

### PR DESCRIPTION
**Pull request by eduardogspereira**
## Context

This PR add the new color palette that was created by @lucas-hidalgo. The palette can be found [here](https://www.figma.com/file/3GCJip7FTCcOAC1A4e2wGk/Colors).

For the onboarding screen that will be constructed (https://github.com/pagarme/pilot/issues/1579) we need this new colors as variables.

## Checklist
- [x] Add new color palletes

## Linked Issues
- [x] Issue https://github.com/pagarme/credenciamento/issues/463

### CODE REVIEW

File: src/styles/colors/light.css [hunk 1]
```diff
@@ -82,6 +82,37 @@
       var(--color-light-salmon-100) 90%
     );
 
-  --color-barney-purple-100: #8267d3;
-  --color-barney-purple-200: #6045af;
+  --color-barney-purple-20: #dfd5ff;
+  --color-barney-purple-50: #8267d3;
+  --color-barney-purple-100: #6045af;
```
Comment (rsmelo) [inline src/styles/colors/light.css]: vi que:
- `--color-barney-purple-100` agora é #6045af e antes era #8267d3
- `--color-barney-purple-200` agora é #3f2787 e antes era #6045af

tem que mudar essas duas mesmo?

Comment (eduardogspereira) [reply src/styles/colors/light.css]: hmm, boa pergunta @rsmelo. Vou ver com o @lucas-hidalgo :)

Comment (lucas-hidalgo) [reply src/styles/colors/light.css]: É isso mesmo, @rsmelo

Comment (rsmelo) [reply src/styles/colors/light.css]: beleza @lucas-hidalgo @eduardogspereira

**Pull request merged**